torch/_dynamo/eval_frame.py

```diff
@@ -608,6 +609,10 @@ def export(
         tracing_mode (str): Specifies the tracing mode. Must be set to "real" if decomposition_table is not specified.
         If decomposition_table is specified, the options are "symbolic" or "fake". Default is "real".
 
+        constraints List[Constraint]: A list of user specified constraints, see Note - [On Export Dynamic Dimension UX]
+        for more details. Constraints must be unique per tensor+dim pair, must not be called alongside mark_dynamic
+        for the same tensor, and must only specify tensors that are part of the arg inputs.
```

Comment:
Also, please learn how to properly format docblocks. Read the ReST manual or something, or go look at some of our existing docblocks.